### PR TITLE
blogpost draft about EPPD criteria

### PR DESCRIPTION
**Pull request by missytake**
There are still images missing, and it's hard to think of any... maybe a screenshot of the table as it stands? Or an unsplash.com image of some servers?

### CODE REVIEW

File: _posts/2021-09-11-eppd-provider-criteria.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+---
+title: What Makes a Good E-Mail Provider?
+author: missytake, jette
+image: ../assets/blog/2021-05-html-mail.jpg
+---
+
+
+What provider should I use for Delta Chat? I hear this question so often that
```
Comment (hpk42) [inline _posts/2021-09-11-eppd-provider-criteria.md]: maybe better use "we" form generally

File: _posts/2021-09-11-eppd-provider-criteria.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+there should be something in the [FAQ](help). But it's not so easy to answer.
+Everyone has different needs. So how to make this decision?
```
Comment (hpk42) [inline _posts/2021-09-11-eppd-provider-criteria.md]:
```suggestion
Everyone has different needs. So how to go about the issue of recommending providers?
```

**Comment by Simon-Laux** [comment]
maybe add emojis or icons to list items to make it more pleasant to read? and also some images, currently it feels too much like a technical wall of text.

File: _posts/2021-09-11-eppd-provider-criteria.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+We can now evaluate providers after these criteria and publish the answers on
+[providers.delta.chat](https://providers.delta.chat). We will start with 20
+popular providers from different countries, and aim to add more in the future.
+
+There were some other criteria which were too complicated to measure, but still
+interesting: e.g. where the servers of a provider are based, and with which
+jurisdiction it has to comply; or whether they collaborated with governments in
+the past, as it just caused an outcry in a [case against climate
+activists](https://protonmail.com/blog/climate-activist-arrest/). Unfortunately
+we can't really use this information for the database.
```
Comment (Simon-Laux) [inline _posts/2021-09-11-eppd-provider-criteria.md]: that sentence somehow triggers a "Why?" question in my head, even though the answer "too complicated to measure" is provided above, maybe you can find more clear wording for it.

File: _posts/2021-09-11-eppd-provider-criteria.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+1. How much delay is between sending and receiving a message through various providers? 
+2. Does a provider systematically block e-mails from other (smaller or larger) providers? 
+3. How many recipients per message are allowed  (limits group size)?
+4. How many messages per hour are accepted until a rate limit is reached and messages are rejected?
+5. What is the max. attachment size for a message?
+6. Does the provider send quota warnings and efficient means for removing messages from server?
+7. Does the provider support SMTP and IMAP and has a valid TLS?
+8. Does the provider support IDLE /MOVE/ IMAP and quota extension?
+9. Does the provider support alias and/ or plus extension?
```
Comment (Simon-Laux) [inline _posts/2021-09-11-eppd-provider-criteria.md]: might need examples and/or explanation for normal people to understand.

**Pull request merged**